spotify/main.py

```diff
@@ -272,8 +298,8 @@ def main():
             },
         )
 
-        start = dt.datetime.now() - dt.timedelta(days=30)
-        end = dt.datetime.now()
+        start = start_datetime - dt.timedelta(days=30)
```

Comment:
do we need to hardcode the days here, isn't it just the defined start date?

Reply:
We have a longer range of 30 days (so the last month) in this case. Don't we want to keep that?

I'm thinking it could be

```python
end = end_datetime
start = end_datetime - dt.timedelta(months=1)
```

Same for the case below.

What do you think?

Reply:
we receive one value per day, so 30days are not really needed. we can also use the 3 days default here and import a bigger range in an import batch process